src/components/UploadCSVView/ReadOnlyRow/ReadOnlyRow.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDisclosure } from '@chakra-ui/react';
+import './ReadOnlyRow.css';
+import EditIcon from '../../../assets/edit.png';
+import DeleteIcon from '../../../assets/delete.png';
+import GreenCheckIcon from '../../../assets/green-check.png';
+import RedCrossIcon from '../../../assets/red-x.png';
+import DeleteBoxModal from '../DeleteBoxModal/DeleteBoxModal';
+
+const ReadOnlyRow = ({ data, editRow, handleDeleteRow }) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const onDelete = () => {
+    handleDeleteRow(data.id);
+    onClose();
+  };
+
+  return (
+    <tr key={data.id}>
```

Comment:
Could we use Chakra tables here instead? https://chakra-ui.com/guides/integrations/with-react-table